components/lib/crash/metrics.yaml

```diff
@@ -18,7 +18,8 @@ crash_metrics:
     labels:
       - uncaught_exception
       - caught_exception
-      - native_code_crash
+      - fatal_native_code_crash
+      - nonfatal_native_code_crash
     bugs:
       - https://bugzilla.mozilla.org/1553935
     data_reviews:
```

Comment:
Nit: also add a reference to this data review

Reply:
Understood.  Thanks!